src/main/kotlin/no/nav/familie/ks/sak/kjerne/behandling/steg/vilkårsvurdering/domene/VilkårResultat.kt

```diff
@@ -95,6 +95,9 @@ class VilkårResultat(
 
     fun erIkkeAktuelt() = this.resultat == Resultat.IKKE_AKTUELT
 
+    fun harFullBarnehageplass() =
+        this.søkerHarMeldtFraOmBarnehageplass == true && this.antallTimer == null
```

Comment:
Har man full barnehageplass når timer er null? 🤔

Reply:
Denne blir litt feil, men har full barnehageplass når antalltimer er høyere enn 32.99timer

Reply:
Syntes og det var rart, men baserte meg på Sats.hentGyldigSatsFor() . Tanken er nok at dersom vi ikke angir hvor mange timer barnet er i barnehagen så går vi utifra at de er der "fulltid". Kan legge til at vi også godtar antallTimer >= 33 

 when {
            erDeltBosted -> 50
            antallTimer == null -> 100
            antallTimer in BigDecimal(0.00)..BigDecimal(8.99) -> 80
            antallTimer in BigDecimal(9.00)..BigDecimal(16.99) -> 60
            antallTimer in BigDecimal(17.00)..BigDecimal(24.99) -> 40
            antallTimer in BigDecimal(25.00)..BigDecimal(32.99) -> 20
            else -> 0
        }

Reply:
100 betyr i dette tilfellet 100% utbetaling

Reply:
Betyr det at vi ikke burde godta at søkerHarMeldtFraOmBarnehageplass == false && antallTimer != null og søkerHarMeldtFraOmBarnehageplass == true && antallTimer == null?

Reply:
Må vel bli no à la `søkerHarMeldtFraOmBarnehageplass == true && antallTimer != null && antallTimer >= BigDecimal(33)`

Reply:
> when {
erDeltBosted -> 50
antallTimer == null -> 100
antallTimer in BigDecimal(0.00)..BigDecimal(8.99) -> 80
antallTimer in BigDecimal(9.00)..BigDecimal(16.99) -> 60
antallTimer in BigDecimal(17.00)..BigDecimal(24.99) -> 40
antallTimer in BigDecimal(25.00)..BigDecimal(32.99) -> 20
else -> 0
}

100 i dette tilfellet betyr jo 100% utbetaling, som man får når man **_ikke_** har barnehageplass